documentation/commands/README.md

```diff
@@ -1,3 +1,475 @@
-# Cg commands
+# cg commands
 This is documentation for cg commands.
 
+## cg add
+Add new things to statusDB.
+
+### cg add customer
+Add a new customer with a unique INTERNAL_ID and NAME.
+
+```$ cg add customer CUSTOMER_ID CUSTOMER_NAME```
+
+where `CUSTOMER_ID` our internal customer ID, e.g. cust000, and `CUSTOMER_NAME` is the customer name.
```

Comment:
CUSTOMER_NAME is like a given name (Kenny Billiau) or a customer description (CMMS)?